chia/rpc/wallet_rpc_client.py

```diff
@@ -677,6 +677,21 @@ async def add_uri_to_nft(self, wallet_id, nft_coin_id, key, uri, fee):
         response = await self.fetch("nft_add_uri", request)
         return response
 
+    async def nft_calculate_royalties(
+        self,
+        royalty_assets_dict: Dict[Any, Tuple[Any, uint16]],
+        fungible_asset_dict: Dict[Any, uint64],
+    ) -> Dict[Any, List[Dict[str, Any]]]:
+        request: Dict[str, Any] = {
+            "royalty_assets": [
+                {"asset": id, "royalty_address": royalty_info[0], "royalty_pts": royalty_info[1]}
```

Comment:
Can we rename `royalty_pts` to `royalty_percentage` to be consistent with the existing field in nft_info?